google/cloud/security/inventory/pipelines/load_projects_buckets_acls_pipeline.py

```diff
@@ -78,40 +76,29 @@ def _retrieve(self):
         Raises:
             LoadDataPipelineException: An error with loading data has occurred.
         """
+        buckets_acls = []
+
         # Get the projects for which we will retrieve the buckets.
         try:
-            project_numbers = self.dao.get_project_numbers(
-                self.PROJECTS_RESOURCE_NAME, self.cycle_timestamp)
+            raw_buckets = self.dao.get_raw_buckets(
+                'buckets', self.cycle_timestamp)
```

Comment:
can we replace the hardcoded 'buckets' string with a variable for code style consistency?